### ISSUE TITLE
val_loss is nan

### ISSUE THREAD

**Issue by allipilli-harshitha** [AntonMu/TrainYourOwnYOLO#116]
Hi AntonMu, 
**Question 1:**
i have a dataset of 208 images.
in which 188 for training and 20 for testing.
from the first epoch onwards i am getting nan as val_loss.
i have changed filters=27 and classes=4 in cfg file.

**Question 2:**
all the images are in the size of 500 x 500 while annotating. so in data_train.txt file, xmax value will be 500 in some cases. But in train_yolo.py, the input shape is given as 416 x 416. does this make any problem?

**Comment by allipilli-harshitha** [comment]
@AntonMu i am getting nan values for val_loss 
I have changed anchors according to my data set using kmeans.py in cfg file
classes=4, filters=27, width=height=608
in yolo_train.py, i have changed batchsize=1, earlystopping(patience=20), input_shape=608,608, 
model.compile(optimizer=Adam(lr=1e-10)

in cfg file i have changed last yolo layers mask values from 0,1,2 to 1,2,3.


before changing those parameters also i got val_loss as nan and even after changing alsi i am getting val_loss as nan
pls help me out with this issue

![WhatsApp Image 2020-04-29 at 5 42 49 PM](https://user-images.githubusercontent.com/60619885/80602383-ae060880-8a4c-11ea-92db-f748d49c8285.jpeg)
